src/Compilers/CSharp/Portable/Symbols/Source/SourceMemberContainerSymbol_ImplementationChecks.cs

```diff
@@ -1259,6 +1259,21 @@ private bool IsValidOverrideReturnType(Symbol overridingSymbol, TypeWithAnnotati
                 }
             }
 
+            TypeWithAnnotations getNotNullIfNotNullOutputType(TypeWithAnnotations outputType, ImmutableHashSet<string> notNullIfParameterNotNull)
+            {
+                for (var i = 0; i < overriddenParameters.Length; i++)
+                {
+                    var overridingParam = overridingParameters[i + overridingMethodOffset];
+                    var overriddenParam = overriddenParameters[i];
+                    if (notNullIfParameterNotNull.Contains(overridingParam.Name) && !overriddenParam.TypeWithAnnotations.NullableAnnotation.IsAnnotated())
```

Comment:
>!overriddenParam.TypeWithAnnotations.NullableAnnotation.IsAnnotated() [](start = 84, length = 69)

Are we handling cases where the overridden parameter type is an unconstrained type parameter or in a `#nullable disable` context?

Reply:
`DelegateCreation_FromMethodGroup_NotNullIfNotNull_02` has a scenario when the overridden input parameter is oblivious. Adding a test for unconstrained type parameters.